test/nodes/test_connector.py

```diff
@@ -129,52 +135,40 @@ def test_crawler_output_file_structure(test_url, tmp_path):
 
 
 def test_crawler_filter_urls(test_url, tmp_path):
-    crawler = Crawler(output_dir=tmp_path)
+    crawler = Crawler(output_dir=tmp_path, save_file_path_on_meta=True)
 
-    paths = crawler.crawl(urls=[test_url + "/index.html"], filter_urls=["index"], crawler_depth=1)
-    assert len(paths) == 1
-    assert content_match(crawler, test_url + "/index.html", paths[0])
+    documents = crawler.crawl(urls=[test_url + "/index.html"], filter_urls=["index"], crawler_depth=1)
+    assert len(documents) == 1
+    assert content_match(crawler, test_url + "/index.html", documents[0].meta["file_path"])
```

Comment:
Should we also test `file_path_meta_field_name` is actually used when passed to the constructor?